src/SearchResults.js

```diff
@@ -0,0 +1,185 @@
+    } else {
+      setIsActive(isActive => (isActive = i));
+    }
+  };
+  const getId = id => {
+    setIndex(id);
+  };
+
+  return (
+    <div>
+      <table className="table">
+        <thead>
+          <tr>
+            <th scope="col">
+              <button
+                className="btn btn-primary"
+                type="button"
+                onClick={() => requestSort("title")}
+              >
+                Title
+              </button>
+            </th>
+            <th scope="col">
+              <button
+                className="btn btn-primary"
+                type="button"
+                onClick={() => requestSort("firstName")}
+              >
+                Firstname
+              </button>
+            </th>
+            <th scope="col">
+              <button
+                className="btn btn-primary"
+                type="button"
+                onClick={() => requestSort("surname")}
+              >
+                Surname
+              </button>
+            </th>
+            <th scope="col">
+              <button
+                className="btn btn-primary"
+                type="button"
+                onClick={() => requestSort("email")}
+              >
+                Email
+              </button>
+            </th>
+            <th scope="col">
+              <button
+                className="btn btn-primary"
+                style={{ fontSize: "10px", padding: "30px" }}
+                type="button"
+                onClick={() => requestSort("roomId")}
+              >
+                Room id
+              </button>
+            </th>
+            <th scope="col">
+              <button
+                className="btn btn-primary"
+                type="button"
+                onClick={() => requestSort("checkInDate")}
+              >
+                Check in date
+              </button>
+            </th>
+            <th scope="col">
+              <button
+                className="btn btn-primary"
+                type="button"
+                onClick={() => requestSort("checkOutDate")}
+              >
+                Check out date
+              </button>
+            </th>
+            <th scope="col">
+              <button className="btn btn-primary">staying</button>
+            </th>
+            <th scope="col">
+              <button className="btn btn-primary"> Customer Profile</button>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {items.map((item, i) => {
+            const {
+              id,
+              title,
+              firstName,
+              surname,
+              email,
+              roomId,
+              checkInDate,
+              checkOutDate
+            } = item;
+            var a = moment(checkOutDate);
+            var b = moment(checkInDate);
+            return (
+              <tr
+                style={
+                  isActive === i ? { background: "yellow" } : { background: "" }
```

Comment:
It's great that you've got the highlighting working, but if you read the instructions carefully it says that the rows should be _independent_. That means you should be able to highlight 2 rows at once.

Can you think of a way of fixing that?

Reply:
Hi Ali, I have found some documents on the internet for multiple selection over table but that's seems quite complicated to understand. Is there any document that you can suggest for me to have a look ?
Thank you.